payload/Library/Scripts/Install or Defer.sh

```diff
@@ -3,18 +3,18 @@
 
 ###
 #
-#            Name:  install_or_defer.sh
+#            Name:  Install or Defer.sh
```

Comment:
Computers usually handle spaces in filenames fine, but this raises the bar for admins who are new to shell and want to test the script. It also means we have to be extra careful to quote paths that reference the script. (All look good here.)

Reply:
Good flag, and thanks for confirming all the path references in the script are correctly quoted. If it's raised as a concern later, we can add a note to the README as a general reminder.